src/shared/components/home/home.tsx

```diff
@@ -529,22 +519,14 @@ export class Home extends Component<HomeRouteProps, HomeState> {
   }
 
   get mySidebar() {
-    const {
-      siteRes: {
-        site_view: { local_site, site },
-        admins,
-      },
-    } = this.state;
+    const { siteRes } = this.state;
 
     return (
       <div id="sidebarContainer">
         <SiteSidebar
-          site={site}
-          admins={admins}
-          localSite={local_site}
+          site={siteRes.site_view.site}
+          siteRes={siteRes}
```

Comment:
Since this is a case where site sidebar is also used on the community page, I'd prefer you pass `activePlugins` as a prop to it. 

In general with components (to avoid rerendering), you want to pass as little as possible (and move up a level if every caller needs the same data), and especially don't pass potentially redundant / overlapping fields like this one now does.